sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -993,7 +996,10 @@ class JDBCSuite extends SparkFunSuite
         Seq(StructField("NAME", StringType, true), StructField("THEID", IntegerType, true)))
       val df = sql("select * from people_view")
       assert(df.schema.size === 2)
-      assert(df.schema === schema)
```

Comment:
revert it back. 

Change the following line https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala#L309 
to
```
fields(i) = StructField(columnName, columnType, nullable)
```

You also need to update some test cases due to the above change, I think.

Reply:
We shouldn't change it, because `scale` is used to infer column types: https://github.com/apache/spark/blob/v2.2.0/sql/core/src/main/scala/org/apache/spark/sql/jdbc/OracleDialect.scala#L32